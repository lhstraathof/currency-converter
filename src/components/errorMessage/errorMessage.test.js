import { render, screen } from '@testing-library/react';
import { create } from 'react-test-renderer';
import { ErrorMessage } from '@components';

const testData = {
	title: 'title',
	error: 'error',
};

describe('ErrorMessage', () => {
	it('renders correctly', () => {
		render(<ErrorMessage title={testData.title} error={testData.error} />);
		expect(screen.getByText(testData.title)).toBeInTheDocument();
		expect(screen.getByText(testData.error)).toBeInTheDocument();
	});

	it('matches snapshot', () => {
		const component = create(<ErrorMessage title={testData.title} error={testData.error} />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
