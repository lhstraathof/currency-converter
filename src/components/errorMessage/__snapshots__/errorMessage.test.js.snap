// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage matches snapshot 1`] = `
<div
  className="error"
>
  <h2
    className="h2"
  >
    title
  </h2>
  <p
    className="p"
  >
    error
  </p>
</div>
`;
